checker/src/callbacks.rs

```diff
@@ -183,6 +183,6 @@ impl rustc_driver::Callbacks for MiraiCallbacks {
             }
             info!("done with analysis");
         });
-        false // Do not proceed to code generation. MIRAI is only a checker.
+        true // Although MIRAI is only a checker we still need code generation for build scripts.
```

Comment:
Is there any danger here if people WANTED to use MIRAI and generate code for full projects? As in, before this change they could run MIRAI and then code generation as separate steps. Now that MIRAI will generate code, it would make more sense to do it in one go. Is that safe?

Reply:
Sadly, it takes so long that nobody would want to do it. It is kind of safe in the sense that MIRAI does not make any changes to the MIR so the generated code is not affected, for better or worse, by the analysis itself. It is not quite safe in the sense that mirai-annotations behave differently when the code is compiled with MIRAI. Right now that only means that ([debug_]checked_)(verify|assume|precondition) become calls to dummy methods that do nothing at runtime.

In the future there may be some more differences as model state and trait contracts come into being.